tests/test_browser.py

```diff
@@ -81,6 +86,21 @@ def test_no_custom_firefox_path(self):
         browser_kwargs_tuple = bok_choy.browser._local_browser_class('firefox')  # pylint: disable=protected-access
         assert 'firefox_binary' not in browser_kwargs_tuple[2]
 
+    @patch.dict(os.environ, [('SELENIUM_FIREFOX_LOG', '/foo/file.log')])
+    def test_custom_firefox_log(self):
+        browser_kwargs_tuple = bok_choy.browser._local_browser_class('firefox')  # pylint: disable=protected-access
+        assert 'firefox_binary' in browser_kwargs_tuple[2]
+
+    @patch.dict(os.environ, [('SELENIUM_FIREFOX_LOG', '')])
+    def test_no_custom_firefox_log(self):
+        browser_kwargs_tuple = bok_choy.browser._local_browser_class('firefox')  # pylint: disable=protected-access
+        assert 'firefox_binary' not in browser_kwargs_tuple[2]
+
+    @patch.dict(os.environ, [('SELENIUM_FIREFOX_PATH', '/foo/path'), ('SELENIUM_FIREFOX_LOG', '/foo/file.log')])
+    def test_custom_firefox_path_and_log(self):
+        browser_kwargs_tuple = bok_choy.browser._local_browser_class('firefox')  # pylint: disable=protected-access
+        assert 'firefox_binary' in browser_kwargs_tuple[2]
```

Comment:
Looks like there are now 4 different tests with the same content; break out into a utility function?  Should there be further assertions on the FirefoxBinary object attributes to make sure it's configured correctly for each scenario?